x-pack/plugins/file_upload/common/constants.ts

```diff
@@ -5,22 +5,31 @@
  * 2.0.
  */
 
-export const UI_SETTING_MAX_FILE_SIZE = 'fileUpload:maxFileSize';
-
-export const MB = Math.pow(2, 20);
-export const MAX_FILE_SIZE = '100MB';
-export const MAX_FILE_SIZE_BYTES = 104857600; // 100MB
-
-export const ABSOLUTE_MAX_FILE_SIZE_BYTES = 1073741274; // 1GB
-export const FILE_SIZE_DISPLAY_FORMAT = '0,0.[0] b';
+const UI_SETTING_MAX_FILE_SIZE = 'fileUpload:maxFileSize';
+const MB = Math.pow(2, 20);
+const MAX_FILE_SIZE = '100MB';
+const MAX_FILE_SIZE_BYTES = 104857600; // 100MB
+const ABSOLUTE_MAX_FILE_SIZE_BYTES = 1073741274; // 1GB
+const FILE_SIZE_DISPLAY_FORMAT = '0,0.[0] b';
 
 // Value to use in the Elasticsearch index mapping meta data to identify the
 // index as having been created by the ML File Data Visualizer.
-export const INDEX_META_DATA_CREATED_BY = 'file-data-visualizer';
+const INDEX_META_DATA_CREATED_BY = 'file-data-visualizer';
 
-export const FILE_FORMATS = {
+const FILE_FORMATS = {
   DELIMITED: 'delimited',
   NDJSON: 'ndjson',
   SEMI_STRUCTURED_TEXT: 'semi_structured_text',
   // XML: 'xml',
 };
+
+export const FILE_UPLOAD = {
```

Comment:
it looks like only ML's DFA is an external consumer of these constants now (and I'm not sure why it even needs `INDEX_META_DATA_CREATED_BY`?)
I don't think we need to export them externally at all anymore and could remove this `FILE_UPLOAD` wrapper.
Having a wrapper like this is not something I've seen before in other plugins.

Reply:
Thanks for the feedback, refactored the exports in 8e81067d21e and a1c0658611d now like this:

- `plugins/file_upload/common` only exports what's consumed by other plugins.
- The plugin itself directly imports the files `common/types|constants.ts` when importing via `public|server`.
- That let me get rid again of that `FILE_UPLOAD` wrapper, no other plugins consume now any constants, only types.